### DISCUSSION TITLE
更新4.23版本后用默认的minitouch连接不上模拟器，无反应，用ADB input可以连上，但隔一段时间会导致明日方舟卡死，无法重连

### DISCUSSION THREAD

**Discussion by SakuraMonster1** [MaaAssistantArknights/MaaAssistantArknights#6164] [Q&A]
更新4.23版本后用默认的minitouch连接不上模拟器，无反应，用ADB input可以连上，但隔一段时间会导致明日方舟模拟器卡死，无法重连，之前用的4.21版本没有这个问题

**Comment by ABA2396** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/6165 去这里带上日志发
